tests/unit/jasmine-browser.json

```diff
@@ -0,0 +1,20 @@
+{
+  "srcDir": "dist",
+  "srcFiles": [
+    "test-deps.js"
+  ],
+  "specDir": "dist",
+  "specFiles": [
+    "test.js"
+  ],
+  "helpers": [],
+  "env": {
+    "stopSpecOnExpectationFailure": false,
+    "stopOnSpecFailure": false,
+    "random": true
+  },
+  "browser": {
+    "name": "firefox"
```

Comment:
It would be great to keep Chrome in here as well, as having our tests pass in both browsers is definitely something I'd like to keep if possible.

If the config only supports one browser at a time, would it be possible to create two config files perhaps, one for each browser?